src/common/markdown.ts

```diff
@@ -14,3 +14,27 @@ export const markdown: MarkdownIt = MarkdownIt({
     return str;
   },
 });
+
+const defaultRender =
+  markdown.renderer.rules.link_open ||
+  function (tokens, idx, options, env, self) {
+    return self.renderToken(tokens, idx, options);
+  };
+
+markdown.renderer.rules.link_open = function (tokens, idx, options, env, self) {
+  const targetIndex = tokens[idx].attrIndex('target');
+  if (targetIndex < 0) {
+    tokens[idx].attrPush(['target', '_blank']);
+  } else {
+    tokens[idx].attrs[targetIndex][1] = '_blank';
+  }
```

Comment:
I see you use this structure twice. Can you turn it into a function, please?
Something like `setTokenAttribute`.
It will make it much more readable as well.